lib/lynis/lynistest.pm

```diff
@@ -0,0 +1,289 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Base module for Lynis test cases
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#78224
+
+package lynis::lynistest;
+
+use strict;
+use warnings;
+use testapi;
+use utils;
+use autotest;
+
+use base 'consoletest';
+use LTP::TestInfo 'testinfo';
+
+our @EXPORT = qw(
+  $testdir
+  $f_position_b
+  $f_position_c
+  @lynis_ok
+  @lynis_fail
+  @lynis_softfail
+  $lynis_baseline_file
+  $lynis_baseline_file_default
+  $lynis_audit_system_current_file
+  $lynis_audit_system_error_file
+  parse_lynis_section_list
+  load_lynis_section_tests
+  parse_lynis_section_content
+  compare_lynis_section_content
+);
+
+our $testdir = "/tmp/";
+
+# File pointer
```

Comment:
would you please provide more description or comment about the usage of file pointer?

Reply:
The file pointer will save the current file access position before close the file, then next time we will start with the position we saved (not start to read/write at the file's beginning ) to avoid duplication.

Reply:
It is interesting, so it means you will not read/write the file completely in one round , right? you will read/write them separatey.